engine/src/test/scala/cromwell/services/KeyValueServiceActorSpec.scala

```diff
@@ -0,0 +1,186 @@
+package cromwell.services
+
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.ConfigFactory
+import cromwell.CromwellTestkitSpec.TestWorkflowManagerSystem
+import cromwell.engine.WorkflowSourceFiles
+import cromwell.engine.backend.local.LocalBackend
+import cromwell.engine.db.slick.SlickDataAccess
+import cromwell.services.KeyValueServiceActor._
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class KeyValueServiceActorSpec extends FlatSpec with Matchers with CromwellServicesTest {
+  val workflowManagerSystem = new TestWorkflowManagerSystem
+  val actorSystem = workflowManagerSystem.actorSystem
+  val dataAccess = new SlickDataAccess(ConfigFactory.parseString(
+    s"""
+       |db.url = "jdbc:hsqldb:mem:$${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
+       |db.driver = "org.hsqldb.jdbcDriver"
+       |driver = "slick.driver.HsqldbDriver$$"
+       |schema.manager = slick
+     """.stripMargin
+  ))
+  val localBackend = LocalBackend(actorSystem)
```

Comment:
I think this is necessary because of foreign key constraints in the database? 

Just pointing out that it's weird that for us to test our key/value store, we need to create a backend instance :-/